src/spec/PointerInputSpec.ts

```diff
@@ -0,0 +1,135 @@
+/// <reference path="jasmine.d.ts" />
+
+/// <reference path="TestUtils.ts" />
+
+describe('A pointer', () => {
+
+   var engine: ex.Engine = null;
+   var pointers: ex.Input.Pointers = null;
+
+   beforeEach(() => {
+      engine = TestUtils.engine();
+      engine.start();
+      pointers = new ex.Input.Pointers(engine);
+      pointers.init();
+   });
+
+   afterEach(() => {
+      engine.stop();
+   });
+   
+   it('should exist', () => {
+      expect(ex.Input.Pointers).toBeDefined();
+      expect(pointers).toBeTruthy();
+   });
+   
+   it('should fire mousedown events', () => {
+      var eventLeftFired = false;
+      var eventRightFired = false;
+      var eventMiddleFired = false;
+      
+      pointers.primary.on('down', function(ev: ex.Input.PointerEvent) {
+         if (ev.button === ex.Input.PointerButton.Left) {
+            eventLeftFired = true;
+         }
+         if (ev.button === ex.Input.PointerButton.Right) {
+            eventRightFired = true;
+         }
+         if (ev.button === ex.Input.PointerButton.Middle) {
+            eventMiddleFired = true;
+         }
+      });
+
+      var mousedownEventLeft = document.createEvent('MouseEvent');
```

Comment:
Can we create a little helper function at the top of the file to reduce the duplication here?